### PR TITLE
Add DigiByte (DGB)

### PR DESCRIPTION
**Pull request by kolya182**

### CODE REVIEW

File: coin/coins.go [hunk 1]
```diff
@@ -501,6 +502,16 @@ var Coins = map[uint]Coin{
 		MinConfirmations: 0,
 		SampleAddr:       "nano_1trqphog5noig7z888asnjejcie8z1iopxyepcjdo1atps8whxiuwd51ehbw",
 	},
+	DGB: {
+		ID:               20,
+		Handle:           "digibyte",
+		Symbol:           "DGB",
+		Name:             "DigiByte",
+		Decimals:         8,
+		BlockTime:        15000,
```
Comment (hewigovens) [inline coin/coins.go]: what's the block time of DGB?

**Comment by hewigovens** [comment]
Test failed

File: coin/coins.go (hunk 1 above)
Comment (vikmeup) [reply coin/coins.go]: Sounds like 20 seconds or so

Comment (kolya182) [reply coin/coins.go]: 15 seconds https://www.digibyte.co/digibyte-global-blockchain

**Pull request merged**